.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+
+python:
+  - 3.6
+addons:
+  postgresql: "9.4"
+services: postgresql
+env:
+  - DJANGO=2.1.7 DB=postgresql
+
+before_install:
+  - export DJANGO_SETTINGS_MODULE=config.settings
```

Comment:
Same for here on exporting environment variables.

Reply:
Done